### PR TITLE
Monitoring feature

### PR DESCRIPTION
**Pull request by dannon**
Monitoring work, buttons to each aspect

### CODE REVIEW

**Comment by wm75** [comment]
@nekrut @bgruening automated runs are also producing consensus FASTA sequences, which may be worth mentioning.

File: genomics/global_platform/README.md [hunk 1]
```diff
@@ -305,9 +205,173 @@ Allelic-variants (AVs) with maximum allele frequency <80% overlapping with codon
...
+Genome surveillance projects at national levels like COG-UK produce sequencing data for unprecedented numbers of samples. To demonstrate that our system can satisfy the analysis needs of such projects, we are performing near real-time analysis of COG-UK sequencing data as it is being submitted to the European Sequence Archive (ENA). We set up an [automated analysis system](https://github.com/simonbray/ena-cog-uk-wfs) that runs our ARTIC AV-calling workflow and the reporting workflow programmatically on all new COG-UK ARTIC paired-end data via Galaxy's openly accessible API. The system also handles data organization into Galaxy histories and exports of resulting datasets. The resulting AV information is available through the [CRG Viral Beacon](https://covid19beacon.crg.eu/) project (see below for FTP access).
+
+<iframe width="100%" height="1000" frameborder="0"
+  src="https://observablehq.com/embed/@spond/sars-cov-2-cog-uk?cell=*"></iframe>
+
+### Processed COG-UK data
+
+Results of our analyses are distributed as following datasets:
+
+- BAM - deduplicated and realigned data. Only properly paired read paired with mapping quality of at least 20 are retained.
```
Comment (wm75) [inline genomics/global_platform/README.md:219]: Maybe instead:

are distributed as:

- publicly accessible Galaxy histories documenting the complete analysis performed for each sample and providing tabular variant report summaries for analysis batches

  URLs to these histories and the TSV datasets are listed in the JSON file [`currently_processed.json`](https://github.com/galaxyproject/SARS-CoV-2/tree/master/data/cog-uk-tracking) containing an up-to-date list of all analyzed datasets. Specifically, the tabular variant reports are part of the history listed under `report` in the JSON file and a direct link to a per-sample variant report for each batch can be found under `report` -> `datamonkey_link`.

- key result files deposited on a publically readable FTP server. These files include:
  - BAM - deduplicated and realigned data. Only properly paired read paired with mapping quality of at least 20 are retained
  - VCF - variant calls produced with `lofreq` and annotated with `SnpEff`
  - consensus FASTA - consensus sequences for each sample constructed from the called variants, in which a consensus allele is defined by an intra-sample allele frequency of >= 0.75 and in which ambiguous sites with a variant allele reaching an allele frequency between 0.25 and 0.75 are soft-masked.
  
  Use FTP client software such as [FileZilla](https://filezilla-project.org/) to access this data at `ftp://xfer13.crg.eu/`.

**Comment by bgruening** [comment]
Loading this entire page is resource-consuming. @dannon can we somehow indicate that parts of the page are still loading?

**Comment by wm75** [comment]
@nekrut @dannon and, yes, I'm seeing a similar issue like @bgruening when trying to view the page. It makes the fan of my laptop spin like crazy and freezes the browser for quite some time though eventually it finished loading for me.

**Comment by bgruening** [comment]
Splitting it up in multiple pages might help?

**Comment by dannon** [comment]
I'll take a look at the performance of the page now.

**Comment by dannon** [comment]
@bgruening @wm75 I built a quick component to defer loading until the frame is scrolled to -- this seems to help dramatically on my end but can you test in your browser as well?  If this is a viable approach I'll polish it up.

**Comment by bgruening** [comment]
Yes, way better. Is it possible to show a spinner as long as it loading? And can we get rid of the scrollbars? Many of them are not scrollable, on my screen at least.

grafik

Thanks @dannon!

File: .vuepress/components/deferredIframe.vue [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    },
+    methods: {
+        loadMe() {
+            this.loadNotebook = true;
+        },
+    }
+};
+</script>
+<style lang="stylus">
+    // TODO: a spinner or something other than a button, and refactor these styles to be generally applicable.
```
Comment (bgruening) [inline .vuepress/components/deferredIframe.vue:38]: haha, ok, I see ... sorry should have looked at this before I submitted my comment :)

Comment (dannon) [reply .vuepress/components/deferredIframe.vue:38]: @bgruening :)  Did you even see the button, or did the intersection observer work and load the iframe as soon as it scrolled into view?

Comment (bgruening) [reply .vuepress/components/deferredIframe.vue:38]: I never saw the button but a blank field. A spinner during that time would be nice I think - to indicate that something is happening.

Comment (bgruening) [reply .vuepress/components/deferredIframe.vue:38]: grafik

Comment (bgruening) [reply .vuepress/components/deferredIframe.vue:38]: But I guess this is not possible with all the iframe magic.

Comment (dannon) [reply .vuepress/components/deferredIframe.vue]: @bgruening Yeah, we'd have to add the spinner on the observable side :/  I can put one on the preload placeholder but it goes away almost immediately once observable starts displaying.

Comment (bgruening) [reply .vuepress/components/deferredIframe.vue]: Is that something that @spond could add?
Not a big deal I hope.

File: genomics/global_platform/README.md [hunk 3]
```diff
@@ -6,126 +6,35 @@ title: Global platform
...
-| Reporting          | [![](https://img.shields.io/badge/Workflow-Import-red)](https://usegalaxy.org/workflows/imp?id=2967ef82911f2cca) | [![](https://img.shields.io/badge/Workflow-Import-red)](https://usegalaxy.eu/workflows/imp?id=0d10c137a0f08bca) | [![](https://img.shields.io/badge/Workflow-Import-red)](https://usegalaxy.org.au/workflows/imp?id=8e2b94c6fbf44368) |
+As [Fig. 1](#figure-1) shows it works like this:
 
-Click on the <kbd>+</kbd> to import workflow into your account:
+1. You run an appropriate analysis [workflow](#table-1). This generates variant list tables;
+2. You then use Jupyter or Observable notebooks to interpret these tables;
 
-<div class="custom-block tip">How to import workflow:<br><br>
-	<img src="figs/workflow_import_gif.gif" style="align:middle" /></div>
+These workflows allow one to identify allelic variants in SARS-CoV-2 genome with frequencies ranging from 5% to 100%.
```
[the remark this answers is not in the document]
Comment (mvdbeek) [reply genomics/global_platform/README.md:31]: Is it clear that we're talking about intrahost frequencies, not global frequencies ?
```suggestion
These workflows allow one to identify allelic variants in SARS-CoV-2 sequencing data with Variant Allele Frequencies ranging from 5% to 100%. 
```

**Pull request merged**